front-page.php

```diff
@@ -206,6 +239,120 @@
 				</div>
 			</div>
 			<?php endif; ?>
+			<?php if($hasEvents): ?>
+			<div class="homepage__events">
+				<div class="homepage__events__background"></div>
+				<div class="row homepage__events__meta">
+					<div class="col-md-6 col-sm-12">
+						<h2 class="subheader homepage__events__subheader"><?php echo __($fieldValues->featured_events_title)?></h2>
+					</div>
+					<div class="col-md-6 col-sm-12 homepage__events__cta">
+						<a href="/events" class="btn btn--small btn--dark"><?php echo __($fieldValues->featured_events_cta_text); ?></a>
+					</div>
+				</div>
+				<div class="row homepage__events__grid">
+					<?php 
+						if(is_array($fieldValues->featured_events)) {
+
+							foreach($fieldValues->featured_events as $featured_event) {
+								if($featured_event['single_event']) {
+									$event = EM_Events::get(array('post_id' => $featured_event['single_event']->ID, 'scope'	=>	'all'));
+									$event = array_shift(array_values($event));
+						
+									include(locate_template('plugins/events-manager/templates/template-parts/single-event-card.php', false, false));
+								} 
+							}
+						}
+					?>
+					<div class="col-lg-4 col-md-6 events__column homepage__events__count">
+						<?php 
+							$eventsTotal = count(EM_Events::get());
+							if ($eventsTotal > 15 && $eventsTotal <= 105):
+								$eventsTotal = floor(($eventsTotal / 10)) * 10;
+								$eventsTotal .= '+';
+							elseif ($eventsTotal > 105 && $eventsTotal <= 1005):
+								$eventsTotal = floor(($eventsTotal) / 100) * 100;
+								$eventsTotal .= '+';
+							elseif ($eventsTotal > 1005): 
+								$eventsTotal = floor(($eventsTotal / 1000) * 1000);
+								$eventsTotal .= '+';
+							else: 
+								$eventsTotal = '10+';
+							endif;
+						?>
+						<p>
+							<span class="large-number homepage__events__count__span"><?php echo __($eventsTotal) ?></span>
+							<?php echo __('More Events.')?>
+							<a href="/events" class="homepage__events__count__link"><?php echo __($fieldValues->featured_events_secondary_cta_text) ?></a>
+						</p>
+					</div>
+				</div>
+			</div>
+			<?php endif; ?>
+			<?php if(isset($fieldValues->featured_activities)): ?>
+			<div class="homepage__activities">
+				<div class="homepage__activities__background"></div>
+				<div class="row homepage__activities__meta">
+					<div class="col-md-6 col-sm-12">
+						<h2 class="subheader homepage__activities__subheader"><?php echo __($fieldValues->featured_activities_title)?></h2>
+					</div>
+					<div class="col-md-6 col-sm-12 homepage__events__cta">
+						<a href="/activities" class="btn btn--small btn--dark"><?php echo __($fieldValues->featured_activities_cta_text); ?></a>
```

Comment:
Should do a check to see if these fields have values before rendering to screen.

This doesn't need to be wrapped in __() function.  This should only be used for static text